.eslintrc.js

```diff
@@ -79,5 +79,8 @@ module.exports = {
                 ],
             },
         ],
+
+        // customization of rules
+        "react/prop-types": "off", // we let Flow/TS keep track of valid props
```

Comment:
This plugin doesn't understand conditional props.  We were getting around this by setting default values while destructuring.  I think it's better to let the static type checker worry about this.  There are some files in webapp that aren't using Flow yet so we may want to figure out how to enable this rule but only for those files.